src/preamble_minimal.js

```diff
@@ -202,6 +202,8 @@ var wasmOffsetConverter;
 
 #if EXIT_RUNTIME
 
+/**@suppress {duplicate, undefinedVars}*/var dynCall_v;
+/**@suppress {duplicate, undefinedVars}*/var dynCall_vi;
```

Comment:
Hmm, I believe this will regress code size on Closure. It is not smart enough to remove duplicates and unuseds on top level global scope ( https://github.com/google/closure-compiler/issues/3191 )

Does it work if you annotate on the actual location where the function calls take place? If Closure is complaining these are not present, then I believe we are not emitting the appropriate dynCalls in the build.

Reply:
I don't know.    Is this not something we do in other places?   I'm basically try to say its OK for these symbols to not exist..  is there a correct/better way to do that?

Reply:
OK I figured it out.. they belong in src/closure-externs/closure-externs.js

PTAL.